### ISSUE TITLE
Fix PHP parse error with autoload_static script

### ISSUE THREAD

**Issue by jacobarriola** [ZeekInteractive/heisenberg#22]
Hosts (WP Engine, for instance) that don't offer support for PHP 5.6.x are getting a syntax error.

```
vendor/composer/autoload_static.php ...         
PHP Parse error:  syntax error, unexpected '.', expecting ')' in - on line 10        
```

**Comment by jacobarriola** [comment]
@AaronHolbrook any chance we can get some traction on this ticket? 🙏

**Comment by AaronHolbrook** [comment]
(For future reference): This is the exact issue explained in detail: http://dotjoe.tumblr.com/post/145010045326/composer-autoloadstaticphp-error-under-php-55

**Comment by jacobarriola** [comment]
Fixed w #24
